doc/getting_started/setup_fpga.md

```diff
@@ -446,9 +446,9 @@ bazel test --copt=-g --test_output=streamed //sw/device/tests:uart_smoketest_fpg
 Then a connection between OpenOCD and GDB may be established with:
 ```sh
 cd $REPO_TOP
-./bazelisk.sh build --config=riscv32 //sw/device/tests:uart_smoketest_prog_fpga_cw340.elf
+bazel build --config=riscv32 //sw/device/tests:uart_smoketest_prog_fpga_cw340.elf
 riscv32-unknown-elf-gdb -ex "target extended-remote :3333" -ex "info reg" \
-  "$(./bazelisk.sh outquery --config=riscv32 //sw/device/tests:uart_smoketest_prog_fpga_cw310.elf)"
+  "$(bazel outquery --config=riscv32 //sw/device/tests:uart_smoketest_prog_fpga_cw310.elf)"
```

Comment:
`outquery` is only available as a `./bazelisk.sh` subcommand.

Also, is it not safer to get people to use `./bazelisk.sh` by default to avoid Bazel version problems.

Reply:
@HU90m is right, `outquery` is a custom thing. Because of that I actually think it would be better to get people to use `./bazelisk.sh` directly of relying on some bash magic to call `bazel` (which is not even calling bazel but bazelisk...)

Reply:
I did the other way around and replaced `bazel` for`bazelisk.sh`.